static/elements/chromedash-process-overview.js

```diff
@@ -138,6 +142,18 @@ class ChromedashProcessOverview extends LitElement {
          </tr>
        `)}
      </table>
+
+    ${Object.keys(this.progress).length ? html`
+      <chromedash-callout
+        cue="progress-checkmarks" targetid="progress-header" signedin
+        .dismissedcues=${this.dismissedcues}>
```

Comment:
Is the `.` in front of this line intentional?

Reply:
Yes, that sets the property of the chromedash-callout element without going through an attribute.